### PR TITLE
ado integration

### PR DESCRIPTION
**Pull request by SANTOSH-MWP**
[https://mavenwave.atlassian.net/jira/software/c/projects/MC/boards/365?modal=detail&selectedIssue=MC-164](url)

### CODE REVIEW

File: app-config.yaml [hunk 1]
```diff
@@ -99,7 +96,11 @@ scaffolder:
   # see https://backstage.io/docs/features/software-templates/configuration for software template options
 
 catalog:
-  locations: []
+  locations:
+    - type: url
+      target: https://dev.azure.com/santoshhkumarr/_git/REASSERT?path=/ado-templates/template.yaml
+      rules:
+        - allow: [Template]
```
Comment (fosterm-mw) [inline app-config.yaml:103]: Is this repo public? otherwise this will not work without your sandbox's `AZURE_TOKEN`

**Review by fosterm-mw — COMMENTED** [review]
Take a quick look to see if the repo is public, otherwise looks good!

File: app-config.yaml (hunk 1 above)
Comment (fosterm-mw) [reply app-config.yaml:103]: Looks private to me

Comment (SANTOSH-MWP) [reply app-config.yaml:103]: Public , share you with separately

Comment (fosterm-mw) [reply app-config.yaml:103]: Thank you

**Comment by fosterm-mw** [comment]
LGTM

**Review by fosterm-mw — APPROVED** [review]

**Pull request merged**